qemu/tests/cfg/netkvm_log_check.cfg

```diff
@@ -0,0 +1,13 @@
+- netkvm_log_check:
+    virt_test_type = qemu
+    type = netkvm_log_check
+    only q35
```

Comment:
We are sure this test case is not support pc machine type,right?

Reply:
Not really, the i440fx machine type also needs to support it. I have deleted it now.